tools/executors/figmaRes/figma/generateReactIcon.js

```diff
@@ -100,7 +100,7 @@ module.exports = async function generateReactIcon(name, svgCode, customStyles) {
     const colors = getColors(colorIdx);
 
     return `
-import { FC } from 'react';
+
```

Comment:
We use scripts to generate the icons automatically, you may need to re-run the script to update them.

```suggestion
```

See https://github.com/toeverything/AFFiNE/blob/e3e37413930eb0a52f1b030f2d04d049152d003f/docs/how-to-auto-download-figma-assets-in-affine.md